domains/meerkats.shays.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "SHays24",
+        "email": "shays24@student.sacs.nsw.edu.au"
+    },
+    "record": {
+        "CNAME": "the-meerkats-projects-shays24-90484013ec4c0067187cd3a154c706ad6.gitlab.io",
```

Comment:
Cannot have a CNAME in conjunction with other records and vice versa.

Reply:
Thanks, was trying to fix some issues, wasn't sure if that was valid but thought i'd try it.